src/GADM.jl

```diff
@@ -122,4 +123,24 @@ function get(country, subregions...)
     throw(ArgumentError("feature not found"))
 end
 
+"""
+    coordinates(country, levels...)  
+
+Returns a deep array of Float64 coordinates for the requested region.  
+Input: ISO 3166 Alpha 3 Country Code, and further full official names of subdivisions  
+
+## Examples  
+  
+```julia
+coordinates("IND") # Returns array of coordinates of India
+coordinates("IND", "Uttar Pradesh") # Returns array of state Uttar Pradesh
+coordinates("IND", "Uttar Pradesh", "Lucknow") # Returns array of coordinates of district Lucknow
+```
+"""
+function coordinates(country, levels...)
+    p = get(country, levels...)
+    c = GeoInterface.coordinates(p)
+    string(ArchGDAL.getgeomtype(p)) === "wkbMultiPolygon" ? c : [c]
```

Comment:
Isn't there a simpler check with `p isa GeoInterface.MultiPolygon ? c : [c]`

Reply:
Also, notice that the names of the arguments have changed in the master branch to `country` and `subregions...`

Reply:
I got `ArchGDAL.IGeometry` so the type doesn't tell us about the type of polygon. I didn't find any other workaround for this.